src/commonMain/kotlin/Foxy.kt

```diff
@@ -8,9 +8,13 @@ import io.ktor.client.request.*
 import io.ktor.client.statement.*
```

Comment:
It also appears the copy header is missing. Not blocking, but please be sure to include it.

Reply:
Done